src/diffusers/loaders.py

```diff
@@ -288,7 +288,7 @@ def save_function(weights, filename):
         model_to_save = AttnProcsLayers(self.attn_processors)
 
         # Save the model
-        state_dict = model_to_save.state_dict()
+        state_dict = {"unet": model_to_save.state_dict()}
```

Comment:
this would break a bit with the existing format no? E.g. already trained LoRAs have a different serialization format at the moment I think